.changeset/rich-goats-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-insights-backend': patch
```

Comment:
Very much a breaking change, so `minor` change since we're on 0.x along with prefixing the changeset message with `**BREAKING**: `